src/net_processing.cpp

```diff
@@ -2773,6 +2777,25 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             if (fRelay) pfrom.m_relays_txs = true;
         }
 
+        if (greatest_common_version >= WTXID_RELAY_VERSION && m_txreconciliation) {
+            // Per BIP-330, we announce reconciliation support if:
+            // - protocol version per the VERSION message supports for WTXID_RELAY
```

Comment:
0d7823a20650ef3b5f2c2da6b33f6dd142636c38
```suggestion
            // - protocol version per the VERSION message supports WTXID_RELAY
```